cli/cli.cpp

```diff
@@ -0,0 +1,150 @@
+#include <iostream>
+#include <string>
+#include <exception>
+#include <stdexcept>
+#include "display.h"
+
+void help() {
+    std::cout << "Rotate and rearrange displays\n";
+    std::cout << "\n";
+    std::cout << "Arguments:\n";
+    std::cout << "\n";
+    std::cout << "  No args     Show this help, same as /?.\n";
+    std::cout << "  /?          Show this help page.\n";
+    std::cout << "  [rotate | /r] [angle] [display]\n";
+    std::cout << "              Rotate the display with the given number clockwise by the given angle.\n";
+    std::cout << "              The angle a must be a multiple of 90 (or 0 to reset any rotation).\n";
+    std::cout << "  [position | /p] [x] [y] [display]\n";
+    std::cout << "              Rearrange the display with the given number to the given coordinates.\n";
+    std::cout << "              Display must be at least 1 and angle a multiple of 90 (or 0 to reset).\n";
+    std::cout << "  [info | /i] [display]\n";
+    std::cout << "              Display information such as name, orientation and position of displays.\n";
+    std::cout << "              If display argument is not speicfied it shows the info for all displays.\n";
+    std::cout << "\n";
+    std::cout << "Display numbers for all arguments start with 1. \n";
+    std::cout << "Unless otherwise specified display argument may be left out with default value of 1.\n";
+    std::cout << "\n";
+}
+
+int rotate(int argc, char** argv) {
+    if (argc < 3) {
+        std::cout << "Missing angle, use argument /? for further help\n";
+        return 1;
+    }
+
+    int disp;
```

Comment:
variables should always be initialized.

```suggestion
    int disp{};
```
or 
```suggestion
    int disp{0};
```